lib/src/lib.rs

```diff
@@ -1007,171 +459,164 @@ pub struct ColourCmd {
 }
 
 /// [Line Type 0](https://www.ldraw.org/article/218.html#lt0) comment.
-#[derive(Debug, PartialEq)]
+#[derive(Debug, PartialEq, Clone)]
 pub struct CommentCmd {
     /// Comment content, excluding the command identififer `0` and the optional comment marker `//`.
     pub text: String,
 }
 
+impl CommentCmd {
+    pub fn new(text: &str) -> CommentCmd {
+        CommentCmd {
+            text: text.to_string(),
+        }
+    }
```

Comment:
nit:

```suggestion
    pub fn new(text: impl Into<String>) -> CommentCmd {
        CommentCmd {
            text: text.into(),
        }
    }
```

Reply:
I still see the old code, is that resolved? Or did I get confused by the GitHub once again? :)